README.md

```diff
@@ -1,13 +1,36 @@
 # Cluedo - Mystery Randomizer
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The project was to individually create a website, inspired by the Cluedo game. The user shall be able to click the three decks of cards to randomly pick a killer, a weapon and a room. After that the user shall be able to click the button to 'Reveal the Mystery'. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The project started with creating all the objects for each suspect, weapon and room. All these objects had different values. Then I created arrays of the objects to group them together. After that, I created functions that randomly pick killer, weapon and room for the different cards for every time you click on them. I choosed which information regarding the different objects that should be visible in the DOM by connecting the JavaScript with the HTML code. Finally, I created a button with onclick-function that revealed the mystery after killer, weapon and room were choosen. These steps were quite easy with good help from the instructions and all steps really make sense. 
+
+I styled the page a bit, by changing background for the whole site, changing colors and adding pictures for all objects. I added information to the suspects about favourite weapon and then created a Shuffle-function that randomly choosed the suspect's favourite weapon. It did not went out perfectly, several suspects can have the same favourite weapon and the function that should make it more likely that there actual favourite weapon appeared both as favourite weapon information and in the weapon card did not really worked the way I wanted. I made several tries and then removed it. 
+
+At least I got the loader animation to work in CSS and I got it to appear as wanted with help of JavaScript by adding addEventListener method and timeout function. It was fun getting the loaders to work and I will probably elaborate more with them in the future.
+
+The plan was quite straight forward, following the instructions. For me it was also important to change the design to my own and to elaborate with those settings. 
+
+I hope to in the future learn even more about the relation between different objects and arrays. 
+
+## Learning objectives
+
+- Create and manipulate Objects
+
+- Create Arrays
+
+- Manipulating the DOM using JavaScript
+
+## Tech
+
+- JavaScript
+
+- HTML
+
+- CSS 
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Link to view the site live: https://cluedoshuffle.netlify.app/
+
```

Comment:
Great readme that explains your work progress and the final project. Awesome work with with loaders to the cards!

Reply:
Completely agree!